etc/kayobe/inventory/group_vars/overcloud/selinux

```diff
@@ -0,0 +1,7 @@
+---
+# Configure SELinux in permissive mode when configuring a Rocky Linux 9 host.
+selinux_state: "{{ 'permissive' if ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '9' else 'disabled' }}"
+
+# Do NOT reboot to apply SELinux config changes. Operators must opt-in by
+# changing this variable when applying host configuration.
+disable_selinux_do_reboot: False
```

Comment:
I would prefer to change this only on RL9 hosts to avoid changing behaviour elsewhere.